### DISCUSSION TITLE
Adding solokey v2 as challenge response for unlocking keepass vault

### DISCUSSION THREAD

**Discussion by youpie** [solokeys/solo2#157] [Q&A]
Hello,

I'm trying to set up a keypass vault on fedora linux, I want to use the solokey to unlock the vault but in keepass it doesn't detect my device. Am I doing something wrong or is this just not possible?
![image](https://user-images.githubusercontent.com/37704067/223227121-e7dd1a57-910c-4db4-a476-3886e5c4dadd.png)

**Comment by coelner** [comment]
currently not possible: https://github.com/keepassxreboot/keepassxc/issues/3560
